static/js/compose.js

```diff
@@ -271,6 +272,22 @@ function compose_error(error_text, bad_input) {
 
 exports.compose_error = compose_error;
 
+function compose_info(info_text) {
+    const rendered_info_text = info_text;
+    clearTimeout(compose_info_timeout);
+    $("#compose-send-status")
+        .removeClass(common.status_classes)
+        .addClass("alert-info")
+        .stop(true)
+        .fadeTo(0, 1);
+    $("#compose-error-msg").html(rendered_info_text);
```

Comment:
This line seems kind of conflicting. Would it be a better idea to make a new `span` element (something with id like `compose-info-msg` maybe?) and use that instead?